CONTRIBUTORS.md

```diff
@@ -8,6 +8,8 @@ Add yourself as a contributor to this project as a Markdown link that links your
 
 - [Cristina Padilla](https://github.com/Mama-simba) 💃
 - [Teri Eyenike](https://github.com/terieyenike) 🦘
+- [Debbie Otuagomah](https://github.com/thatgirldorian) 😅
+- # [Victoria Emoka](https://github.com/mohanner-coder) 🐶
```

Comment:
These links are the same as the ones below. Pls remove the duplicates

Reply:
I just cleaned up the duplicates and redid the PR title. Appreciate the feedback! :)